oslabs-beta/ReactRelay
# Expand ts

SUMMARY

convert all TypeScript “any" types to specific types


CHANGES

add 'interfaces' folder to 'renderer/src' folder, containing interfaces that collectively cover all state in the redux store, as well a couple type definitions related to IPC

define types of parameters and return values sent via IPC in 'preload/index.d.ts'

add keys to 'MethodButton' components returned via array mapping from the 'MethodButtonContainer' component


TESTING INFORMATION

1. run “npm run typecheck”
	there should be no errors

2. run “npm run make” to compile code and package it into an application file
3. unzip and run this application file
4. test all application features on a couple projects you’ve previously used for testing
